packages/blade/src/components/Input/PasswordInput/PasswordInput.tsx

```diff
@@ -5,8 +5,8 @@ import { EyeIcon, EyeOffIcon } from '~components/Icons';
 import Box from '~components/Box';
 import { CharacterCounter } from '~components/Form/CharacterCounter';
 import { IconButton } from '~components/Button/IconButton';
-import type { NativeHTMLInputRefProps } from '~src/hooks/useExposeInputRef';
-import { useExposeInputRef } from '~src/hooks/useExposeInputRef';
+import type { BladeRefElement } from '~src/hooks/useBladeInnerRef';
```

Comment:
```suggestion
import type { BladeElementRef } from '~src/hooks/useBladeInnerRef';
```
goes well with `useBladeInnerRef` where the first part defines the name of the variable and then the suffix defines the type of it which in this case is `Ref`

Reply:
Makes sense!